models/order/src/parcel/builder.spec.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { DeliveryItem } from '../delivery-item';
+import { ParcelMeasurements } from '../parcel-measurements';
+import { TrackingData } from '../tracking-data';
+import { TParcel, TParcelGraphql } from './types';
+import { Parcel } from '.';
+
+describe('builder', () => {
+  const defaultParcelSpec = {
+    id: expect.any(String),
+    key: null,
+    measurements: expect.objectContaining({
+      heightInMillimeter: expect.any(Number),
+      lengthInMillimeter: expect.any(Number),
+      widthInMillimeter: expect.any(Number),
+      weightInGram: expect.any(Number),
+    }),
+    trackingData: expect.objectContaining({
+      trackingId: expect.any(String),
+      carrier: expect.any(String),
+      provider: expect.any(String),
+      providerTransaction: expect.any(String),
+      isReturn: expect.any(Boolean),
+    }),
+    items: expect.arrayContaining([
+      expect.objectContaining({
+        id: expect.any(String),
+        quantity: expect.any(Number),
+      }),
+    ]),
+    custom: null,
+    createdAt: expect.any(String),
+  };
+
+  it(
+    ...createBuilderSpec<TParcel, TParcel>(
+      'default',
+      Parcel.random(),
+      expect.objectContaining(defaultParcelSpec)
+    )
+  );
+  it(
+    ...createBuilderSpec<TParcel, TParcelGraphql>(
+      'graphql',
+      Parcel.random(),
+      expect.objectContaining({
+        ...defaultParcelSpec,
+        __typename: 'Parcel',
+      })
+    )
+  );
+
+  it('should allow customization', () => {
+    const id = 'test-id';
+    const key = 'test key';
+    const heightInMillimeter = 123;
+    const carrier = 'Test CARRIER';
+    const quantity = 9001;
+
+    const measurements =
+      ParcelMeasurements.random().heightInMillimeter(heightInMillimeter);
+    const trackingData = TrackingData.random().carrier(carrier);
+    const deliveryItem = DeliveryItem.random().quantity(quantity);
+
+    const parcelMock = Parcel.random()
+      .id(id)
+      .key(key)
+      .measurements(measurements)
+      .trackingData(trackingData)
+      .items([deliveryItem])
+      // .custom(custom)
```

Comment:
We're missing the configuration for this property.

Reply:
After discussing about this property with @antoniolodias, we agree that some more missing models need to be migrated, making the scope of this PR even bigger.

Since this property is not currently used in the MC local test data model we're bringing in this PR, we agreed to keep that property commented by now, in order to unblock this PR and others that depend on this one, and work on it in a [separate task](https://commercetools.atlassian.net/browse/FEC-71).